templates/email/bexley/waste/_sidebar_content.html

```diff
@@ -4,6 +4,12 @@
   detail = report.detail.remove(address);
   IF is_no_collections_missed_enquiry OR is_missed_collection;
     key = 'Bins reported as missed';
+  ELSIF is_delivery_request;
```

Comment:
We need to load waste/_email_data.html in the bexley alert-update templates as well

Reply:
I already do this at the top of `templates/email/bexley/alert-update.html` and `templates/email/bexley/alert-update.txt`, or do you mean somewhere else?

Reply:
Huh, did I forget to push my alerts commit last week?